src/main/java/edu/harvard/iq/dataverse/export/ExportService.java

```diff
@@ -154,7 +154,9 @@ public void exportAllFormats(Dataset dataset) throws ExportException {
             if (releasedVersion == null) {
                 throw new ExportException("No released version for dataset " + dataset.getGlobalId().toString());
             }
-            final JsonObjectBuilder datasetAsJsonBuilder = JsonPrinter.jsonAsDatasetDto(releasedVersion);
+            // We pass settingsService into the JsonPrinter constructor to check the :ExcludeEmailFromExport setting.
+            JsonPrinter jsonPrinter = new JsonPrinter(settingsService);
+            final JsonObjectBuilder datasetAsJsonBuilder = jsonPrinter.jsonAsDatasetDto(releasedVersion);
```

Comment:
Could we keep this as a static reference (JsonPrinter.jsonAsDatasetDto(releasedVersion); and perhaps change the constructor line to just be a static set method and use that here instead? (I suspect it was the warning that 'static methods should be accessed in a static manner' from my IDE that caused me to make this change in the first place...) (I can make a PR to your branch if you want.)

Reply:
@qqmyers sure, please make a PR against my branch. Thanks!